### DISCUSSION TITLE
Change Sensor Voltage from 120 to 240V

### DISCUSSION THREAD

**Discussion by Fionnoch** [OpenEVSE/open_evse#42] [Q&A] [answered]
Hi, 

I recently setup a v5.6 OpenEVSE Advanced kit and am very happy with the features however I am in Europe (Ireland specifically) and the voltage the device is defaulting to is 120V but I know that should be 230/240v. Where can I change the default value or else is this a hardware issue? I know the current sensors are correct as I have checked the values against the what the car is reading which is why I believe this is a default value somewhere. 

I have tried changing the setting to 3 phase (I'm only using 1 phase) but that did not output the correct values. Any help would be greatly appreciated :) 

![image](https://github.com/user-attachments/assets/ff114e7d-99a4-4f23-b684-dfaa14242138)

**Comment by chris1howell** [comment, marked as answer]
Set Service Level to 2 (208-240v). Service Level 1 (120v) or Auto is
incorrect for Europe.

**Reply by Fionnoch** [reply]
Brilliant thank you! such a simple fix but I was trolling though files trying to find it
